src/parser/SparqlParserHelpers.cpp

```diff
@@ -66,6 +66,13 @@ ResultOfParseAndRemainingText<ParsedQuery::Alias> parseAlias(
   return p.parse<ParsedQuery::Alias>(
       input, "alias", &SparqlAutomaticParser::aliasWithouBrackes);
 }
+// ____________________________________________________________________________
+ResultOfParseAndRemainingText<GraphPatternOperation::Bind> parseBind(
+    const std::string& input, SparqlQleverVisitor::PrefixMap prefixMap) {
+  ParserAndVisitor p{input, std::move(prefixMap)};
+  return p.parse<GraphPatternOperation::Bind>(input, "bind",
+                                              &SparqlAutomaticParser::bind);
+}
```

Comment:
When we get more of these, we should also have the abstraction here:
```
auto bind = parseBind(input, prefixMap);
```
becomes
```
auto bind = parseImpl<GraphPatternOperation::Bind, SparqlAutomaticParser::bind>(input, prefixMap, "bind");
```
(the name that is passed to the exception message ("bind" in this example) could also be a template parameter, but that is a design decision where I don't have a strong opinion.

Note:
`parseExpression` above currently cannot be fully replaced by this as it has to convert from `SparqlExpression::Ptr` to `SparqlExpressionPimpl`, but it will still lose half of its code, and we can at some point let the Antlr parser return the `Pimpl` directly (but not in this PR).